.travis.yml

```diff
@@ -8,7 +8,7 @@ jdk:
   - oraclejdk8
 
 env:
-  - DOCKER_COMPOSE_VERSION=1.14.0
+  - DOCKER_COMPOSE_VERSION=1.14.0 WHEREHOWS_DIR=wherehows-web
```

Comment:
Let's do one line per env var for readability

Reply:
I initially had it as one line, but travis splits each env line into a matrix, so the build will run twice: 1 for each env var
https://docs-staging.travis-ci.com/user/environment-variables/#Defining-Multiple-Variables-per-Item

https://docs-staging.travis-ci.com/user/customizing-the-build/#Build-Matrix

which will actually cause the build to fail, because the vars are mutually exclusive